### PR TITLE
[WIP/hacky]: Kafka Source tests debugging

### PR DESCRIPTION
**Pull request by Cali0707**
## Proposed Changes

- Log out more info in the tests
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: vendor/knative.dev/reconciler-test/pkg/feature/feature.go [hunk 1]
```diff
@@ -258,6 +258,19 @@ func DeleteResources(ctx context.Context, t T, refs []corev1.ObjectReference) er
 		return true, nil
 	})
 	if err != nil {
+		gv, _ := schema.ParseGroupVersion("eventing.knative.dev/v1alpha1")
```
Comment (pierDipi) [inline vendor/knative.dev/reconciler-test/pkg/feature/feature.go:261]: Group is
https://github.com/knative-extensions/eventing-kafka-broker/blob/828b81ede7af938cdffd01f6f103992469330728/control-plane/config/eventing-kafka-broker/100-kafka-internal/100-consumergroup.yaml#L24

```suggestion
		gv, _ := schema.ParseGroupVersion("internal.kafka.eventing.knative.dev/v1alpha1")
```

**Comment by Cali0707** [comment]
@pierDipi I think we need to run more of the tests, it seems like the bug doesn't occur when running only KafkaSourceDelete tests

**Pull request closed without merging**